types/modules/channels.d.ts

```diff
@@ -0,0 +1,32 @@
+import type amqp = require('amqplib');
+
+interface ChannelConfig {
+  prefetch: number;
+}
+
+declare class Channels {
+  constructor(connection: amqp.Connection, config: ChannelConfig);
+  _connection: any;
+  _config: any;
+  _channels: Map<any, any>;
+  get(queue: any, config: any): Promise<any>;
```

Comment:
can we type the any here? queue is a string, `Promise<any>` should be amqp chnanel